### ISSUE TITLE
Set lyse host to localhost and enable analysis by default

### ISSUE THREAD

**Issue by rpanderson** [labscript-suite/blacs#72]
Another works-out-of-the-box proposal. This behaviour would be consistent with runmananger, which assumes blacs is running on localhost and doesn't require any further intervention before submitting shots on a fresh install.

**Comment by chrisjbillington** [comment]
Definitely. I can do this (tomorrow) if you like - it is super trivial.
